app/src/Flashcards/DictionarySearch.js

```diff
@@ -1,6 +1,32 @@
 import * as React from "react";
+import { useState } from "react";
+
+import useApi from "../auth/useApi";
 
 const DictionarySearch = () => {
-  return <input placeholder="search"></input>;
+  const { apiClient } = useApi();
+  const [definition, setDefinition] = useState([]);
+  const [word, setWord] = useState("");
+  const getDefinition = async (event) => {
+    event.preventDefault();
+    const result = await apiClient.getDefinition(word);
```

Comment:
I'd get in the habit of having a different API client for each API.  For instance, you could have `flashcardApiClient`, `dictionaryApiClient`, etc.  That way, each file has a single responsibility, and the components can import only the logic for the API they need to hit, without importing logic they don't need.